eclipse-ankaios/config/startupState.yaml

```diff
@@ -6,16 +6,16 @@ workloads:
       - key: project
```

Comment:
I will likely remove this file from this change, since Ankaios team has another PR open for this change and we ideally want to use the images from eclipse-sdv-blueprints registry, not ladatz